### PR TITLE
Fix Live Plan UI

### PR DESCRIPTION
**Pull request by jessesleeping**
Print an error message when retrieving the plan of a QUEUED/PLANNING query. 

Assigned by @raghavsethi 
#7493

### CODE REVIEW

File: presto-main/src/main/resources/webapp/assets/plan.js [hunk 1]
```diff
@@ -254,6 +253,15 @@ let LivePlan = React.createClass({
             );
         }
 
+        if (isQueryPlanReady(query) == false) {
```
Comment (raghavsethi) [inline presto-main/src/main/resources/webapp/assets/plan.js]: We can just check if the outputStage is null, like we do in other places.

**Comment by jessesleeping** [comment]
Updated :)

**Comment by jessesleeping** [comment]
Updated :)

File: presto-main/src/main/resources/webapp/assets/plan.js [hunk 2]
```diff
@@ -299,6 +312,12 @@ let LivePlan = React.createClass({
                         { this.renderProgressBar() }
                     </div>
                 </div>
+                <hr className="h3-hr"/>
```
Comment (raghavsethi) [inline presto-main/src/main/resources/webapp/assets/plan.js]: By the way, this line is a regression - adds an extra line to the UI for no reason, and makes it inconsistent with the other pages.

**Comment by raghavsethi** [comment]
Merged, thanks!

**Pull request closed without merging**